src/Dapr.PluggableComponents.Tests/DaprPluggableComponentsApplicationTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿// ------------------------------------------------------------------------
+// Copyright 2023 The Dapr Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+using Xunit;
+
+namespace Dapr.PluggableComponents;
+
+public sealed class DaprPluggableComponentsApplicationTests
+{
+    [Fact(Timeout = 10000)]
```

Comment:
Can we make this 10000 and other such `Timeout` initializations come from a single constant that somehow states its unit? I am assuming this to be 10s but IMBW.

Reply:
Done.